changelog.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v7.1.0...HEAD)
+### Fixed
+* [PCM-1668](https://inindca.atlassian.net/browse/PCM-1668) - Installed newer version of headset library that will help with some issues that were found as well as offer a better approach to rejecting calls.
```

Comment:
I believe the release/v7.2.0 -> master PR does make that change. But since this was needed before we completely merged that PR to master, it appears as if we skipped over the version.  If release/7.2.0 built properly though, we should have the 7.2.0 tag so we could update it now right?

Reply:
We can fix it when we cut the next release.